enhancements/update/versioning-cincinnati-api.md

```diff
@@ -0,0 +1,116 @@
+---
+title: versioning-cincinnati-api
+authors:
+- "@pratikmahajan"
+  reviewers:
+- @openshift/openshift-team-cincinnati-maintainers
+- @openshift/openshift-team-cincinnati
+- @openshift/team-cincinnati-operator
+  approvers:
+- @crawford
+- @LalatenduMohanty
+- @sdodson
+  creation-date: 2021-08-20
+  last-updated: 2021-08-20
+  status: implementable
+  see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# Versioning cincinnati api and json schema
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+
+## Summary
+As of today, we declare the schema version in Cincinnati Graph URI. We currently, do not embed the version information
+in json schema. The enhancement will embed the version information in the json schema. This has the benefit of being 
+easy to implement and to pass around if we ship Cincinnati JSON over non-HTTP channels. Enhancement will also implement
+media type versioning while requesting the graph e.g. `ContentType: application/vnd.redhat.cincinnati.v1+json`. 
```

Comment:
nit: "while requesting the graph" would be [the `Accept` header][1].  I like that content negotiation, but for the non-HTTP channel shipping angle, the header is [`Content-Type`][2] (with a hyphen).  Not sure which way you want to go here.  Maybe just step back and say:

> The enhancement will also begin versioning Cincinnati payloads by media type, for example `application/vnd.redhat.cincinnati.v1+json`.

[1]: https://datatracker.ietf.org/doc/html/rfc7231#section-5.3.2
[2]: https://datatracker.ietf.org/doc/html/rfc7231#section-3.1.1.5